modules/docs/arrow-docs/docs/docs/arrow/typeclasses/align/README.md

```diff
@@ -0,0 +1,60 @@
+---
+layout: docs
+title: Align
+permalink: /docs/arrow/typeclasses/align/
+redirect_from:
+  - /docs/typeclasses/align/
+---
+
+## Align
+
+{:.beginner}
+beginner
+
+The `Align` typeclass extends the `Semialign` typeclass with a value empty(), which acts as a unit in regards to align.
+
+### Main Combinators
+
+#### empty()
+
+returns an empty structure which can be used as either argument for align.
+
+`fun <A> empty(): Kind<F, A>`
+
+```kotlin:ank
+import arrow.core.extensions.*
+import arrow.core.extensions.listk.align.align
+import arrow.core.*
+
+ListK.align().run {
+    align(listOf("A", "B").k(), empty<String>())
+}
+```
+
+### Laws
+
+Arrow provides [`AlignLaws`][functor_laws_source]{:target="_blank"} in the form of test cases for internal verification of lawful instances and third party apps creating their own Align instances.
+
+#### Creating your own `Align` instances
+
+Arrow already provides Align instances for common datatypes (e.g. Option, ListK, SetK). See their implementations
```

Comment:
SetK does not have an instance :see_no_evil:

Reply:
right, thank you. i remember. Set is not a Functor :D